experiments/promoter/promote/1-promote-apis.md

```diff
@@ -0,0 +1,8 @@
+1. Make sure the directory apis/${SERVICE}/v1beta1 exist. If not, create one.
+2. Copy the go files from apis/${SERVICE}/v1alpha1 to apis/${SERVICE}/v1beta1.
+3. Make sure the go files under apis/${SERVICE}/v1beta1 have package name `v1beta1`. For example, you can run run `grep -rl "package v1alpha1" apis/<SERVICE>/v1beta1/ | xargs -i sed "s/package v1alpha1/package v1beta1/g"`
+4. Iterate the new files and change the code `v1alpha1` to `v1beta1` if it represents `apis/${SERVICE}/v1alpha1`
+3. Remove file apis/${SERVICE}/v1beta1/zz_generated.deepcopy.go
```

Comment:
We seem to have the file (Eg. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/v1.132.0/apis/apigee/v1beta1/zz_generated.deepcopy.go) why are we removing here?